tests/test_20_use_case_topology.py

```diff
@@ -515,16 +515,114 @@ def test_070_port_uni_up_no_l2vpn_associated(self):
         final_data = requests.get(API_URL).json()
         assert final_data[l2vpn_id] == l2vpn_data['data'], "L2VPN state changed unexpectedly"
 
-    def test_100_vlan_range_change(self):
+    @pytest.mark.xfail(reason="The L2VPN with VLAN 1060 remains up even after modifying the VLAN range to [100–200]")
+    def test_106_removing_vlan_uni(self):
```

Comment:
Gretel, it is not clear to me what is the difference from this test 106 and the test `test_100_shrinking_vlan_uni`. Can you please clarify?